.github/actions/setup/action.yml

```diff
@@ -84,7 +84,7 @@ runs:
         zip --version
 
     - name: Install sccache
-      uses: spideroak-inc/sccache-action@main
+      uses: aranya-project/sccache-action@main
```

Comment:
Probably needs to be conditional to still work internally

Reply:
If I try to put a conditional value inside the `use`, the linter complains:
`the uses attribute must be a path, a Docker image, or owner/repo@ref`
If I try to use a conditional `if`, the linter also complains:
`Error: Bad request - spideroak-inc/sccache-action@main is not allowed to be used in aranya-project/aranya. Actions in this workflow must be: within a repository that belongs to your Enterprise account, created by GitHub, or matching the following: aranya-project/*.`